### ISSUE TITLE
Don

### ISSUE THREAD

**Issue by stylewarning** [rigetti/quilc#134]

**Comment by notmgsk** [comment]
Would this be enough to satisfy this issue
```diff
-     ("16QMUX" . ,(quil::build-16QMUX-chip))
+     ("16QMUX" . ,(quil::build-nQ-trivalent-chip 1 1 8 4))
      ("bristlecone" . ,(quil::build-bristlecone-chip))
```

**Comment by ecpeterson** [comment]
No: the qubits in `build-16QMUX-chip` follow a different indexing convention than those in `build-nQ-trivalent-chip`. I think an ideal solution would take `width` and `height` options that describe how many octagons there are tiled in each direction. Someone will need to talk to internal sources (and so maybe this isn't a "good first issue") to figure out what the octagon numbering conventions are when there are more than 3 of them.
